root/client/src/components/table/table.js

```diff
@@ -1,29 +1,179 @@
+        }
+        break;
+      case 5:
+        newStateWeek = {
+          'Sunday': new Date().setDate(new Date().getDate() - 5),
+          'Monday': new Date().setDate(new Date().getDate() - 4),
+          'Tuesday': new Date().setDate(new Date().getDate() - 3),
+          'Wednesday': new Date().setDate(new Date().getDate() - 2),
+          'Thursday': new Date().setDate(new Date().getDate() -1),
+          'Friday': today[Symbol.toPrimitive]('number'),
+          'Saturday': new Date().setDate(new Date().getDate() + 1),
+        }
+        break;
+      case 6:
+        newStateWeek = {
+          'Sunday': new Date().setDate(new Date().getDate() - 6),
+          'Monday': new Date().setDate(new Date().getDate() - 5),
+          'Tuesday': new Date().setDate(new Date().getDate() - 4),
+          'Wednesday': new Date().setDate(new Date().getDate() - 3),
+          'Thursday': new Date().setDate(new Date().getDate() - 2),
+          'Friday': new Date().setDate(new Date().getDate() - 1),
+          'Saturday': today[Symbol.toPrimitive]('number'),
+        }
+        break;
+      // Need to change default
+      default: {
+        newStateWeek = {
+          'Sunday': new Date().setDate(new Date().getDate() - 1),
+          'Monday': today[Symbol.toPrimitive]('number'),
+          'Tuesday': new Date().setDate(new Date().getDate() + 1),
+          'Wednesday': new Date().setDate(new Date().getDate() + 2),
+          'Thursday': new Date().setDate(new Date().getDate() + 3),
+          'Friday': new Date().setDate(new Date().getDate() + 4),
+          'Saturday': new Date().setDate(new Date().getDate() + 5),
+        }
+        break;
+      }
+    }
+
+    setWeek(newStateWeek);
+  }, []);
+
+  // 7 * 24 * 60 * 60 * 1000
+
+  const handlePrevWeek = (e) => {
+    e.preventDefault();
+    // Take each day of this.state.week. Move that back 7 days.
+    const newStateWeek = {
+      'Sunday': week['Sunday'] - (7 * 24 * 60 * 60 * 1000),
+      'Monday': week['Monday'] - (7 * 24 * 60 * 60 * 1000),
+      'Tuesday': week['Tuesday'] - (7 * 24 * 60 * 60 * 1000),
+      'Wednesday': week['Wednesday'] - (7 * 24 * 60 * 60 * 1000),
+      'Thursday': week['Thursday'] - (7 * 24 * 60 * 60 * 1000),
+      'Friday': week['Friday'] - (7 * 24 * 60 * 60 * 1000),
+      'Saturday': week['Saturday'] - (7 * 24 * 60 * 60 * 1000),
+    }
+    setWeek(newStateWeek);
+  }
+
+  const handleNextWeek = (e) => {
+    e.preventDefault();
+
+    const newStateWeek = {
+      'Sunday': week['Sunday'] + (7 * 24 * 60 * 60 * 1000),
+      'Monday': week['Monday'] + (7 * 24 * 60 * 60 * 1000),
+      'Tuesday': week['Tuesday'] + (7 * 24 * 60 * 60 * 1000),
+      'Wednesday': week['Wednesday'] + (7 * 24 * 60 * 60 * 1000),
+      'Thursday': week['Thursday'] + (7 * 24 * 60 * 60 * 1000),
+      'Friday': week['Friday'] + (7 * 24 * 60 * 60 * 1000),
+      'Saturday': week['Saturday'] + (7 * 24 * 60 * 60 * 1000),
+    }
```

Comment:
same comment as above

Reply:
Similar to above, made this change. Thanks